how-to/workspace-platform-starter/client/src/modules/integrations/apps/integration.ts

```diff
@@ -434,7 +434,7 @@ export class AppProvider implements IntegrationModule<AppSettings> {
 				if (Array.isArray(tagFilter.options)) {
 					tagFilter.options.push({
 						value: tag,
-						isSelected: false
+						isSelected: true
```

Comment:
Should these default to true?

Reply:
They do for all other filters, everything is selected to begin with